test/yargs-parser.js

```diff
@@ -567,6 +567,7 @@ describe('yargs-parser', function () {
     it('should load nested options from config file', function () {
       var jsonPath = path.resolve(__dirname, './fixtures/nested_config.json')
       var argv = parser(['--settings', jsonPath, '--nested.foo', 'bar'], {
+        configuration: { 'config-dot-notation': true },
```

Comment:
I don't think this logic should need to change because we don't have keys of the form "foo.bar"?

Reply:
Yes, I see what you mean, but it fails without it. Ok, I'll have to look into this one more, tomorrow or Friday.